Lib/test/test_email/test_message.py

```diff
@@ -1004,6 +1004,31 @@ def test_folding_with_long_nospace_http_policy_1(self):
         parsed_msg = message_from_bytes(m.as_bytes(), policy=policy.default)
         self.assertEqual(parsed_msg['Message-ID'], m['Message-ID'])
 
+    def test_no_wrapping_max_line_length(self):
+        # Test that falsey 'max_line_length' are converted to sys.maxsize.
+        for n in [0, None]:
+            with self.subTest(max_line_length=n):
+                self.do_test_no_wrapping_max_line_length(n)
+
+    def do_test_no_wrapping_max_line_length(self, n):
+        pol = policy.default.clone(max_line_length=n)
```

Comment:
```suggestion
    def do_test_no_wrapping_max_line_length(self, n):
        self.assertFalse(n)
        pol = policy.default.clone(max_line_length=n)
```

We can rename `n` to "falsey" as well. Even if the test is near, someone reading it shouldn't need to read the `test_*` function itself.

Reply:
Sorry, I missed this comment! Yes, that could be a positive change. Thanks!